lesson_06/lesson_06.py

```diff
@@ -0,0 +1,81 @@
+import os
+import platform
+from typing import Optional
+import random
+
+
+OS_slash_sym = "\\"  # default slash symbol for path variable if your OS is Windows
+FILES_AMOUNT = 20
+EXTENSIONS = ["txt", "mp3", "pdf", "docx", "py", "tmp"]
+EMPTY_STRING = ""
+
+
+pc_user_name = str(os.getlogin())  # get pc user name
+print(f"Your PC USER NAME is : {pc_user_name}")
+
+
+default_path = ''.join(("C:", OS_slash_sym, "Users", OS_slash_sym, os.getlogin(), OS_slash_sym, 'Documents'))
```

Comment:
Next time you can try to use this approach (result will be the same, but code will be a bit shorter):
```
from os import path

path.join('one', 'two')
# 'one/two'

```